ansible/roles/distributed_press/tasks/main.yml

```
@@ -10,9 +27,14 @@
     shell: /bin/bash
     comment: User for the distributed.press API server
 
+- name: "Ensure nginx can access site data"
+  shell: gpasswd -a www-data {{distributed_press_user}}
+
```

Remove dependency on `gpasswd` shell using built in function

```suggestion
- name: "Ensure nginx can access site data"
  user:
    name: '{{distributed_press_user}}'
    groups: www-data
    append: yes
```